Source/SwiftLintFramework/Rules/RuleConfigs/NameConfig.swift

```diff
@@ -14,11 +14,19 @@ public struct NameConfig: RuleConfig, Equatable {
     var excluded: Set<String>
 
     var minLengthThreshold: Int {
-        return max(minLength.warning, minLength.error)
```

Comment:
This can just be `return max(minLength.warning, minLength.error ?? minLength.warning)`